README.md

```diff
@@ -1,3 +1,45 @@
 # Inago
 
 [![Build Status](https://api.travis-ci.org/giantswarm/inago.svg)](https://travis-ci.org/giantswarm/inago) [![codecov.io](https://codecov.io/github/giantswarm/inago/coverage.svg?branch=master)](https://codecov.io/github/giantswarm/inago?branch=master)
+
+Inago is a deployment tool that manages groups of unit files to deploy them to
+a fleet cluster similar to `fleetctl`. Since `fleetctl` is quite limited, Inago
+aims to abstract units away and provide more sugar on top like update
+strategies. That way the user can manage unit files more easily.
+
+## Getting Inago
+
+Clone the git repository: `git@github.com:giantswarm/inago.git`
+
```

Comment:
I oriented on the yochu readme. There is none. Here it is also only calling `make`.